docs/cloudify_script_plugin/conf.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
This isn't actually necessary for the prints we're doing, we can just use the function syntax but not the function itself (unless that breaks on 2.6?)

Reply:
it's not really the function syntax then, it's printing a string which is wrapped in parens for no reason; I'm not gonna do this because I don't like code which does one thing but looks like it does another :P

Reply:
Fair enough, marking resolved.